indico/modules/events/abstracts/templates/abstract/public.html

```diff
@@ -283,27 +283,42 @@ <h3 class="abstract-title">
         <div class="toolbar thin">
             {% if abstract.can_judge(session.user, check_state=true) %}
                 <div class="group">
-                    <a href="#abstract-decision-box" class="i-button highlight icon-hammer">
+                    <a href="#abstract-decision-box" class="i-button text-color highlight icon-hammer">
                         {% trans %}Judge{% endtrans %}
                     </a>
                 </div>
             {% endif %}
-            {% if abstract.event_new.can_manage(session.user) %}
-                {% if management %}
-                    <div class="group">
-                        <a href="{{ url_for('.display_abstract', abstract) }}" class="i-button highlight icon-switchon">
-                            {% trans %}Display view{% endtrans %}
+            {% set can_manage = abstract.event_new.can_manage(session.user) %}
+            {% if abstract.submitter == session.user or can_manage %}
+                <div class="group">
+                    {% if abstract.state.name != 'withdrawn' and (can_manage or abstract.public_state.name == 'awaiting') %}
+                        <a title="{% trans %}Withdraw abstract{% endtrans %}"
+                           data-href="{{ url_for('.withdraw_abstract', abstract, management=management) }}"
+                           data-method="POST"
+                           data-update='{"display_html": ".mainContent .col2", "management_html": ".management-page"}'
+                           data-title="{% trans %}Withdraw Abstract{% endtrans %}"
+                           data-confirm="{% trans %}Do you really want to withdraw this abstract? This operation is irreversible.{% endtrans %}"
+                           class="i-button text-color danger icon-cross">
```

Comment:
We tend to put the class attribute at the top, but there's no stronger argument for it.